apps/frontend/src/renderer/components/ui/resizable-panels.tsx

```
@@ -0,0 +1,137 @@
+      </div>
+
+      {/* Resizable divider */}
+      <div
+        className={cn(
+          "w-1 flex-shrink-0 relative cursor-col-resize",
+          "bg-border transition-colors duration-150",
+          "hover:bg-primary/40",
+          isDragging && "bg-primary/60"
+        )}
+        onMouseDown={handleMouseDown}
+      >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)